src/drive/web/modules/drive/files.js

```diff
@@ -6,6 +6,8 @@ const FILE_TYPE = 'file'
 const DIR_TYPE = 'directory'
 export const ALBUMS_DOCTYPE = 'io.cozy.photos.albums'
 export const isFile = file => file && file.type === FILE_TYPE
+export const isOnlyOfficeFile = file =>
```

Comment:
je pensais qu'on se basait sur la class txt / spreadsheet / slide ? Non ?

Reply:
La class `text`est aussi présente pour un `.txt` et un `.md` qu'on ne souhaite pas prendre en compte avec only office. En creusant un peu je me suis dit qu'avec le mime de cette manière on aurait un test plus pertinent.

Reply:
Pas sûr non. Côté stack le check est fait sur la class, on devrait en faire de même pour avoir le même comportement (je t'ai pingué sur la PR là-bas ce matin) 

Si je regarde https://www.askingbox.com/info/mime-types-of-microsoft-office-file-formats on se rend compte qu'il y a plein de cas ou ce n'est pas `officedocument`

Reply:
Ok pour le mime effectivement 👍 

Ok vu => https://github.com/cozy/cozy-stack/pull/2962#discussion_r611376818
Par contre on doit alors ajouter une exclusion supplémentaire pour les `.txt` et `.md` en fonction de l'extension du fichier.

Reply:
modifié en 
```
export const isOnlyOfficeFile = file =>
  !file.name.endsWith('.txt') &&
  !file.name.endsWith('.md') &&
  (file.class === 'text' ||
    file.class === 'spreadsheet' ||
    file.class === 'slide')
```

?

Reply:
shouldBeOpenedByOnlyOffice() et ça devrait répondre à toutes les questions que tu te poses ?